server/api/auth/email.get.ts

```
@@ -0,0 +1,31 @@
+                statusMessage: 'User not found',
+            })
+        }
+        return { email: user.email }
+    } catch (err: any) {
+        throw createError({
+            statusCode: 500,
+            statusMessage: 'Server error',
+            data: String(err),
+        })
+    }
+    return { email: user.email }
```

Biome (2.1.2)

[error] This code will never be reached ...

... because either this statement will return from the function, ...

... this statement will return from the function, ...

... this statement will return from the function, ...

... or this statement will throw an exception beforehand


(lint/correctness/noUnreachable)